Training4/thongnq-training4/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React, {useState, useEffect, useCallback} from "react";
+
+import axios from "axios";
+
+import Country from "./task2/Country";
+
+import debounce from 'lodash.debounce';
+
+const countriesAPI = "https://restcountries.eu/rest/v2";
+
+const App = () => {
+  const [countries, setCountries] = useState([]);
+
+  const hanldeLoadCountry = (api) => {
+    axios.get(api).then(response => {
+      setCountries(response.data);
+    })
+  }
+
+  const debounceOnChange = useCallback(debounce(hanldeLoadCountry, 1000), []);
```

Comment:
@Thong-Nguyen-Zigvy 
debounceOnChange is created in every the state changes or the parent re-render, it make the debounce run incorrect, please wrap it using useCallback to make sure they created only one